api/helpers/utils.js

```diff
@@ -28,6 +27,38 @@ const strToBool = (str) => {
   return valsHash[found || str.toLowerCase()];
 };
 
+/**
+ * updateJuvenileName
+ * Update the juvenile's name in case no last named was passed with the
+ * parent's last name. The ILS returns names in an array called `names`.
+ * @param {string} name
+ * @param {array} parentArrayName
+ */
+const updateJuvenileName = (name, parentArrayName = []) => {
+  const parentsName = parentArrayName[0];
+  // If there's no parent name, then just return the name for the child.
+  if (!parentsName) {
+    return name;
+  }
+
+  let updatedName = name;
+  // If there's no last name, then use the parent's last name. This is a very
+  // basic check that is done by checking if there is a space in the complete
```

Comment:
But it's possible for someone's first name to be more than one word...

Reply:
Yes, but right now we don't take first or last names as separate request inputs. It's just `"name"`. For now, this is fine but I'll make a ticket to make the request accept first and last name which is a larger update than this which is needed soon.